features/staging_commands.feature

```diff
@@ -197,6 +203,207 @@ Feature: Enable command behaviour when attached to an UA staging subscription
            | xenial  | FIPS Updates | fips-updates |https://esm.staging.ubuntu.com/fips-updates/ubuntu xenial-updates/main |
            | bionic  | FIPS Updates | fips-updates |https://esm.staging.ubuntu.com/fips-updates/ubuntu bionic-updates/main |
 
+    @series.xenial
```

Comment:
I think this needs to be @series.bionic right because we don't expect ubuntu-azure-fips to exist on Xenial FIPS PPAs?